test/e2e/combobox.cy.ts

```diff
@@ -283,16 +283,50 @@ describe("Combobox", () => {
           }
         });
       });
+    });
 
-      context("when losing focus", () => {
-        it("should render empty string on the searchbox", () => {
-          cy.findAllByPlaceholderText("Select a fruit...")
-            .eq(1)
-            .click()
-            .type("{enter}{downarrow}{enter}", { force: true });
-          cy.findAllByPlaceholderText("Select a fruit...").eq(1).click();
-          cy.findByLabelText("textbox-search").should("have.value", "");
-        });
+    context("when losing focus", () => {
+      it("should render empty string on the searchbox", () => {
+        cy.findAllByPlaceholderText("Select a fruit...")
+          .eq(1)
+          .click()
+          .type("{enter}{downarrow}{enter}", { force: true });
+        cy.findAllByPlaceholderText("Select a fruit...").eq(1).click();
+        cy.findByLabelText("textbox-search").should("have.value", "");
+      });
+    });
+  });
+
+  context("With Display", () => {
```

Comment:
can we make sure that after selecting, the text shown in the combobox is the `text` ?